packages/synapse-react-client/.storybook/main.mts

```diff
@@ -60,7 +55,6 @@ const config: StorybookConfig = {
     // Fix deployment to github pages
     if (configType === 'PRODUCTION') {
       base = './'
-      plugins = [turbosnap({ rootDir: config.root ?? process.cwd() })]
```

Comment:
`turbosnap` is now automatically included in the Storybook+Vite integration, so we can remove it from our config